Emby.Server.Implementations/Updates/InstallationManager.cs

```diff
@@ -71,29 +70,24 @@ public class InstallationManager : IInstallationManager
             IApplicationHost appHost,
             IApplicationPaths appPaths,
             IHttpClient httpClient,
-            IJsonSerializer jsonSerializer,
             IServerConfigurationManager config,
             IFileSystem fileSystem,
             IZipClient zipClient,
             IConfiguration appConfig)
         {
-            if (logger == null)
-            {
-                throw new ArgumentNullException(nameof(logger));
-            }
-
             _currentInstallations = new List<(InstallationInfo, CancellationTokenSource)>();
             _completedInstallationsInternal = new ConcurrentBag<InstallationInfo>();
 
             _logger = logger;
             _applicationHost = appHost;
             _appPaths = appPaths;
             _httpClient = httpClient;
-            _jsonSerializer = jsonSerializer;
             _config = config;
             _fileSystem = fileSystem;
             _zipClient = zipClient;
             _appConfig = appConfig;
+
+            _jsonOptions = JsonDefaults.GetOptions();
```

Comment:
It looks like these defaults are used for serialization of data into the Sqlite databases. Maybe it would be a good idea to define a new set of defaults to use for web requests, since it has different requirements. And these new web defaults could also be re-used for configuring serialization in the new ASP endpoints?

Reply:
These were created to mimic the behavior of the old default serializer

Reply:
Again, misunderstood the existing serialization behavior, this is fine